src/consensus/vote.cpp

```diff
@@ -131,6 +132,7 @@ VoteManager::VoteManager(addr_t node_addr, std::shared_ptr<DbStorage> db, std::s
 void VoteManager::addUnverifiedVote(taraxa::Vote const& vote) {
   uint64_t pbft_round = vote.getRound();
   auto hash = vote.getHash();
+  vote.process();  // speed up we do not need to calcualte again
```

Comment:
not needed if we cache stuff as described in above comment

Reply:
it is need because later we work only with copies, so it has to be there

Reply:
so make that copy later when you need to ?